docs/apache.md

```diff
@@ -1,52 +1,11 @@
-# `apache httpd` Metrics Receiver
+# Apache Web Server (httpd)
 
-The Apache httpd receiver can retrieve stats from your Apache web server using the `/server-status?auto` endpoint.
-
-
-## Prerequisites
-
-The receiver requires that you enable the [mod_status module](https://httpd.apache.org/docs/2.4/mod/mod_status.html) on your Apache web server.
-
- This requires adding the following lines to the server’s `httpd.conf` file:
-
-```
-<Location "/server-status">
-    SetHandler server-status
-    Require host localhost
-</Location>
-```
-
-## Configuration
-
-Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your Apache web server configuration.
-
-To configure a receiver for your Apache web server metrics, specify the following fields:
-
-| Field                 | Default                   | Description |
-| ---                   | ---                       | ---         |
-| `type`                | required                  | Must be `apache`. |
-| `server_status_url`     | `http://localhost:8080/server-status?auto` | The url exposed by the `mod_status` module. |
-| `collection_interval` | `60s`                     | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
-
-Example Configuration:
-
-```yaml
-metrics:
-  receivers:
-    apache_metrics:
-      type: apache
-      server_status_url: http://localhost:8080/server-status?auto
-      collection_interval: 30s
-  service:
-    pipelines:
-      apache_pipeline:
-        receivers:
-          - apache_metrics
-```
+Follow https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/apache
+for instructions to collect logs and metrics from this application using Ops Agent.
```

Comment:
Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/apache) to collect logs and metrics from this application using Ops Agent.

Reply:
Sure thing. Changed.